### DISCUSSION TITLE
Let User select 802.11 version in WiFi HotSpot

### DISCUSSION THREAD

**Discussion by 38github** [rerdavies/pipedal#148] [Ideas]
I had a big problem with getting my phone to find PiPedal and it strangely was solved by choosing the old b/g 802.11 specification outside of PiPedal. Is this something that can be implemented in PiPedal and hopefully easier wghen it supports NetworkManager?

**Comment by rerdavies** [comment]
Selecting the wifi channel in the Wi-Fi Direct configuration dialog effectively does that:

- Channels 1-11: 802.11g
- Channels higher than that: 802.11a/n.
- Wifi 6 channels are currently not supported.

Selecting channel 1, 6 or 11 will fix that for you.

This feature has been completely reworked under NetworkManager in v1.2.33. I'm not sure channel selection actually worked in prior versions.

For what it's worth, I have an old Android tablet that only supports g channels. Works fine with the current version of PiPedal.

**Reply by 38github** [reply]
I will try this out! Thank you!
